tools/tune/tdfb/bf_design.m

```diff
@@ -246,6 +266,9 @@
     denom2 = denom1 * W_w;
     di = num / denom2;
     bf.di_db(iw) = 10*log10(abs(di));
+    if isequal(DEBUG_DIAG_LOAD_ON,1)
+        bf.di_db(iw) = log(abs(di)*fs/N)*1e+1/3;
```

Comment:
I still wonder why the di_db calculation needs to change here. Also 10*log10() is nicer to read than log10()*1e+1. What is the divide by 3 for?

Reply:
Sure - Your proposal make sense. Thanks